### ISSUE TITLE
Example from README not working

### ISSUE THREAD

**Issue by gubertoli** [daneoshiga/tapioca-jarbas#8]
```
dr@dr:~$ python3
Python 3.5.2 (default, Nov 23 2017, 16:37:01) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from tapioca_jarbas import Jarbas
>>> jarbas = Jarbas()
>>> receipt = jarbas.receipt(document_id=5920249).get()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/dr/.local/lib/python3.5/site-packages/tapioca/tapioca.py", line 256, in get
    return self._make_request('GET', *args, **kwargs)
  File "/home/dr/.local/lib/python3.5/site-packages/tapioca/tapioca.py", line 230, in _make_request
    data = self._api.process_response(response)
  File "/home/dr/.local/lib/python3.5/site-packages/tapioca/adapters.py", line 60, in process_response
    data = self.response_to_native(response)
  File "/home/dr/.local/lib/python3.5/site-packages/tapioca/adapters.py", line 119, in response_to_native
    return response.json()
  File "/home/dr/.local/lib/python3.5/site-packages/requests/models.py", line 892, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib/python3.5/json/__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.5/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.5/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
>>>
```

**Comment by daneoshiga** [comment]
should be fixed now in 1.0.1, sorry for not seeing this earlier.

**Comment by ffreitasalves** [comment]
This issue should be closed because it works know.
